packages/ddp-client/livedata_connection_tests.js

```diff
@@ -95,6 +95,92 @@ Tinytest.add("livedata stub - receive data", function (test) {
   test.isUndefined(conn._updatesForUnknownStores[coll_name]);
 });
 
+Tinytest.add("livedata stub - buffering data", function (test) {
+  var lolex = Npm.require('lolex');
```

Comment:
Just use `require()` (or `import lolex from 'lolex'` if we are using `ecmascript` in this package/test?)